requirements.txt

```diff
@@ -1,6 +1,8 @@
 httmock
 wooper
 PyRTF3
+# reportlab is fixed due to support for python 3.5
+reportlab==3.5.53
```

Comment:
This makes sense in the `master` branch, when this is ported to the `develop` we probably should bump Python to 3.6+

Reply:
This one caught us out on our last release!